homeassistant/components/switcher_kis/__init__.py

```diff
@@ -12,7 +12,7 @@
 from homeassistant.core import Event, HomeAssistant, callback
 from homeassistant.helpers import device_registry as dr
 
-from .const import DOMAIN
+from .const import CONF_TOKEN, DOMAIN
```

Comment:
This token exists as core constant:
`from homeassistant.const import CONF_TOKEN`

Reply:
It is done.